project_resource_calendar/views/resource_view.xml

```diff
@@ -9,12 +17,7 @@
     </record>
     <menuitem id="resource_calendar_menu_root"
               name="Resource calendar"/>
```

Comment:
@eilst : [Minor comment] Can be changed Resource Calendar instead of Resource calendar. Ignore if not applicable in ur context.